### DISCUSSION TITLE
QRow and QCol not available

### DISCUSSION THREAD

**Discussion by alsa85** [quasarframework/quasar#17241] [General - Components / Directives / etc]
Hi devs, I'm a newbee in the use of quasar and I have a strange issue. I can't use QRow and QCol components because they seems to be missing in my project. I have tried at all, also reinstall dependencies but in node_modules/quasar I don't see anything about this two components. For now I work on a Windows 10, can be this the problem in this case?

**Comment by yusufkandemir** [comment]
From Discord:

> There are no such components. If you are using something like ChatGPT, it might be making them up. I am not sure why you assume they should exist.
See https://quasar.dev/layout/grid/introduction-to-flexbox for actual grid management, which is done through CSS utility classes.
```html
<div class="row">
  <div class="col">
    col1
  </div>
  <div class="col">
    col2
  </div>
</div>
```
> Yes, there are also other classes like `column`, `col-12`, `col-6`, `col-xs-4`, etc. which are explained in the docs

![image](https://github.com/quasarframework/quasar/assets/6266078/527ce83b-7ba8-4ba6-b25a-a0c02c01932f)
![image](https://github.com/quasarframework/quasar/assets/6266078/87b874f0-ed00-43fa-b39e-5c28cb3572d6)
